Assets/PatchKit Patcher/Scripts/AppData/Local/LocalMetaData.cs

```diff
@@ -129,6 +166,12 @@ private bool TryLoadDataFromFile()
             try
             {
                 _data = JsonConvert.DeserializeObject<Data>(File.ReadAllText(_filePath));
+
+#if UNITY_5_3_OR_NEWER // LEGACY: fill productKey from unity prefs
+                _data.productKey = string.IsNullOrEmpty(_data.productKey)
+                    ? UnityEngine.PlayerPrefs.GetString(ValidateLicenseCommand.CachePatchkitKey, string.Empty)
```

Comment:
This is actually a good place to load PlayerPrefs license key data. Just remember to remove it if present.